src/main/java/org/jabref/logic/ai/chatting/chathistory/storages/MVStoreChatHistoryStorage.java

```diff
@@ -119,10 +119,6 @@ private Map<Integer, ChatHistoryRecord> getMap(Path bibDatabasePath, String type
         return mvStore.openMap(bibDatabasePath + "-" + type + "-" + name);
     }
 
-    public void commit() {
-        mvStore.commit();
-    }
-
```

Comment:
I did not see any discussion why a removal of that is OK. @InAnYan ?

Reply:
This is just an overriden method

Reply:
Sometimes, this diff-based reviewing is shitty ^^